src/main/java/com/google/sps/templates/calendar.soy

```diff
@@ -27,20 +30,84 @@
               <span class="mdc-list-item__text"><h2>Event Calendar</h2></span>
             </a>
 
-            <a class="mdc-list-item" href="#">
-              <label class="mdc-text-field mdc-text-field--filled">
-                <span class="mdc-text-field__ripple"></span>
-                <input class="mdc-text-field__input" id="address" type="text" aria-labelledby="my-label-id">
-                <span class="mdc-line-ripple"></span>
-              </label>  
+            <hr class="mdc-list-divider">
+            <div class="nav-icons">
+              <a class="icon-links" href="/chat">
+                <i class="material-icons mdc-list-item__graphic nav-icon">message</i>
+              </a>
+              <a class="icon-links" href="/calendar">
+                <i class="material-icons mdc-list-item__graphic nav-icon">date_range</i>
+              </a>
+              <a class="icon-links" href="/groups">
+                <i class="material-icons mdc-list-item__graphic nav-icon">group</i>
+              </a>
+            </div>
```

Comment:
I've actually already changed this in my branch, but if you want to change it here too, the href in the second <a> tag should be "/map" and the <i> tag below it should be map instead of date_range. also in my branch, my order is map, chat, and groups